quantities/registry.py

```diff
@@ -1,41 +1,49 @@
 """
 """
 
+import ast
 import re
-import builtins
 
 
 class UnitRegistry:
+    # Note that this structure ensures that UnitRegistry behaves as a singleton
 
     class __Registry:
 
         __shared_state = {}
+        whitelist = (
+            ast.Expression,
+            ast.Constant,
+            ast.Name,
+            ast.Load,
+            ast.BinOp,
+            ast.UnaryOp,
+            ast.operator,
+            ast.unaryop,
+            ast.Num,
```

Comment:
```suggestion
```
ast.Num is covered by ast.Constant and will be removed in future versions of Python (according to pylance on vscode for me :). I tested the quantities suite after removing num and it was fine.

Reply:
@apdavison do you want to try deleting this and testing with the whole suite? I tested on 3.11 removing and it was fine, but maybe the ast.Num is necessary for older versions?